update screenshot and 1line for llm obs config doc

barieom opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
<!-- A brief description of the change being made with this pull request. What is your motivation for the PR? -->

### Merge instructions

<!-- 
If you're waiting for a release or there are other considerations that you want us to be aware of, list them here. 
If the PR is ready to be merged once it receives the required reviews, check the box below after you've created the PR.
-->

Merge readiness:
- [x] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

aliciascott commented on content/en/llm_observability/configuration/_index.md:
```diff
@@ -31,7 +31,7 @@ Connect your OpenAI account to LLM Observability with your OpenAI API key.
 1. Follow the instructions on the tile. 
    - Provide your OpenAI API key. Ensure that this key has **write** permission for **model capabilities**.
-1. Enable **Invoke Model from this API key**.
+1. Enable **Use this API key to evaluate your  LLM applications**.
```
```suggestion
1. Enable **Use this API key to evaluate your LLM applications**.
```

aliciascott reviewed (changes requested):
Hi @barieom just one small spacing thing and then can you also keep the original image as it takes some time for the translated pages to update and it could show a broken image there. You'd just want to rename the new one like openai-tile_2.png and update that in the `img src`.

barieom commented:
> Hi @barieom just one small spacing thing and then can you also keep the original image as it takes some time for the translated pages to update and it could show a broken image there. You'd just want to rename the new one like openai-tile_2.png and update that in the `img src`.

Thanks @aliciascott - addressed your comments

aliciascott reviewed (approved):
looks good thank you!

aliciascott commented:
/merge
